packages/amazon-sumerian-hosts-babylon/examples/hello-world-demo.js

```diff
@@ -0,0 +1,49 @@
+import DemoUtils from './common/demo-utils.js';
+import cognitoIdentityPoolId from './common/demo-credentials.js';
+
+let host;
+let scene;
+
+async function createScene() {
+  // Create an empty scene. IMPORTANT: Sumerian Hosts require use of the
+  // right-hand coordinate system!
+  scene = new BABYLON.Scene();
+  scene.useRightHandedSystem = true; // IMPORTANT for Sumerian Hosts!
+
+  DemoUtils.setupSceneEnvironment(scene);
+  initUi();
+
+  // ===== Configure the AWS SDK =====
+
+  AWS.config.region = cognitoIdentityPoolId.split(':')[0];
+  AWS.config.credentials = new AWS.CognitoIdentityCredentials(
+    { IdentityPoolId: cognitoIdentityPoolId },
+  );
+
+  // ===== Instantiate the Sumerian Host =====
+
+  // Edit the characterId if you would like to use one of
+  // the other pre-built host characters. Available character IDs are:
+  // "Cristine", "Fiona", "Grace", "Maya", "Jay", "Luke", "Preston", "Wes"
+  const characterId = 'Cristine';
+  const pollyConfig = { pollyVoice: 'Joanna', pollyEngine: 'neural' };
+  const characterConfig = HOST.HostUtils.getCharacterConfig('./assets/character-assets', characterId);
+  host = await HOST.HostUtils.createHost(scene, characterConfig, pollyConfig);
```

Comment:
It's great that we moved so much critical code out of the sample app, but is something called HostUtils the right place? Should these go directly into HostObject? I'm not quite understanding the split between those. The HostObject looks like a more generic system, and HostUtils creates specific implementations that we know of as Preston/Wes/luke/etc? Should this be called SumerianHost or AmazonHost or something like that?

Reply:
I waffled on this question myself and I'm happy to change it. And let me clarify, the HostUtils class can accommodate custom host characters as well as the "built-in" host characters. It does feel more like it should be a part of HostObject, but I was hesitant to bolt on 350 lines of code to that object. But it's probably a better choice. I'll make that change.

Reply:
This issue is addressed in the update I just posted.